ouroboros-network/src/Ouroboros/Network/BlockFetch/Client.hs

```diff
@@ -79,16 +78,14 @@ blockFetchClient :: forall header block m.
                  => Tracer m (TraceFetchClientEvent header)
                  -> FetchClientPolicy header block m
                  -> FetchClientStateVars m header
-                 -> STM m PeerGSV
```

Comment:
I suppose this can let us close #469?

Reply:
Not yet. This just shuffles things around. We still don't have the GSV measurement system in place yet.

Reply:
I see. But the block fetch client (and thus the consensus layer that initiates it) doesn't have to care about GSV measurements anymore, it is something internal to the block fetch logic, right?